scaletest/terraform/coder.tf

```diff
@@ -197,7 +201,7 @@ resource "local_file" "kubernetes_template" {
           }
           resources {
             requests = {
-              "cpu"    = "0.1"
+              "cpu"    = "0.01"
```

Comment:
This seems low to me --- do we have evidence that agents only consume 0.01 CPUs, even during traffic generation?

We want to avoid agent resource throttling.  If agent resources are the bottleneck during our tests, then we will not be able to find the bottlenecks in Coderd.

Reply:
I only reduce this to be able to fit more agents on a node. Maybe it makes more sense to make these configurable at the Terraform level?

Reply:
I do think it's worth making a variable, but we should set the default to something sensible for a "reasonable" traffic test.